languages/csharp/exercises/concept/properties/.docs/after.md

```diff
@@ -0,0 +1,79 @@
+The two main types of property are 
+
+1. auto-implemented properties where the `get` and `set` accessors have no body.
+They may or may not be explicitly initialized.  For example:
```

Comment:
Formatting pedantry: Is that a newline between the sentences? IIRC the usual formatting for markdown on Exercism is to not introduce newlines unless a paragraph break is intended. If I am correct here, this will apply to a lot of other lines and files in this commit.